sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/CommunicationRelayServiceVersion.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.networktraversal;
+
+import com.azure.core.util.ServiceVersion;
+
+/**
+ * The versions of Communication Relay Service supported by this client library.
+ */
+public enum CommunicationRelayServiceVersion implements ServiceVersion {
+    V2021_03_07("1.0.0b1");
```

Comment:
Please update version to 1.0.0-beta.1

Reply:
Also the V2021_03_07 seems a bit out of date. Please doouble check that it is correct